### PR TITLE
Travis integration. Closes #54

### LINKED ISSUE #54: Configure CI
e.g. Travis, Circle etc.

### PR DESCRIPTION
**Pull request by anxolerd**
Travis integration closes #54
Add CI integration.
Use tox for running tests within different environments

Due to bug in tox there is a workaround with passing envvar to tests command.

-------------
Review request: @webknjaz

### CODE REVIEW

**Comment by webknjaz** [comment]
@anxolerd Plz sync updated `config/test/app.yml` from original branch

File: requirements/dev.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 pre-commit==0.7.6
 -r common.txt
 hg+https://hg@bitbucket.org/webknjaz/blueberrypy-wk#egg=blueberrypy[speedups,dev]
+tox==2.3.1
```
Comment (webknjaz) [inline requirements/dev.txt]: I'd add this to `test.txt` instead

File: config/test/app.yml [hunk 2]
```diff
@@ -57,7 +57,7 @@ controllers:
       tools.sessions.on: true
 
 sqlalchemy_engine:
-  url: <connection uri here>
+  url: "sqlite://"
```
Comment (webknjaz) [inline config/test/app.yml]: Drop quotes

**Comment by anxolerd** [comment]
Moved tox dependency from dev requirements to test requirements in 6e2b48b
Updated testing config from api-for-admin branch in 2a985ea

**Pull request merged**